docs/experiments/taint-labels.md

```diff
@@ -0,0 +1,23 @@
+---
+slug: taint-labels
+append_help_link: true
+description: "Taint labels increase the expressiveness of taint-mode by allowing to specify and track different kinds of taint through labels."
+---
+
+# Taint labels
+
+Previously taint mode could only track one kind of taint, data was either tainted or clean and that had to be specified using no more than a single pattern operator. Sometimes this is not enough. For example, data may become _dangerous_ through several steps that are not easily specified with a single pair source-sink. Taint labels increase the expressiveness of taint-mode by allowing to specify and track different kinds of taint through labels. 
```

Comment:
> Can we say that: `Taint labels allow you to match more tainted sources`.?

I find it a bit confusing to use "match" here. The matching expressiveness didn't change because for that we use the matching engine. We could perhaps say `Taint labels allow you to specify more complex taint sources`.